cmd/admin-revoker/main.go

```diff
@@ -305,6 +564,79 @@ func main() {
 			fmt.Printf("%d: %s\n", k, revocation.ReasonToString[k])
 		}
 
+	case (command == "private-key-block" || command == "private-key-revoke") && len(args) == 2:
+		// 1: keyPath, 2: reasonCode
+		keyPath := args[0]
+
+		reasonCode, err := strconv.Atoi(args[1])
+		cmd.FailOnError(err, "Reason code argument must be an integer")
```

Comment:
Do we need to accept reason code as a parameter? We know it always has to be 1 (keyCompromise).